docs/testing-policies/02-policy-authors.md

```diff
@@ -1,60 +1,46 @@
 ---
-sidebar_label: "Policy Authors"
-title: ""
+sidebar_label: "Policy authors"
+title: "Testing for policy authors"
+description: An introduction to testing Kubewarden policies for policy authors.
+keywords: [kubewarden, policy testing, policy author, rust, go, assemblyscript, development environment]
 ---
 
-# While creating a policy
+Kubewarden policies are regular programs compiled as WebAssembly (Wasm).
+As with any kind of program, good test coverage is important.
 
-Kubewarden Policies are regular programs compiled as WebAssembly. As with any kind
-of program, it's important to have good test coverage.
+Policy authors can use their favorite development environments, tools and, testing frameworks to verify their development.
 
-Policy authors can leverage the testing frameworks and tools of their language
-of choice to verify the behaviour of their policies.
+These three Kubewarden policies provide an example written in each of [Rust](/writing-policies/rust/01-intro-rust.md), [Go](/writing-policies/go/01-intro-go.md) and [AssemblyScript](https://www.assemblyscript.org/) respectively:
 
-As an example, you can take a look at these Kubewarden policies:
+- [psp-apparmor](https://github.com/kubewarden/psp-apparmor)
+- [ingress-policy](https://github.com/kubewarden/ingress-policy)
+- [pod-privileged-policy](https://github.com/kubewarden/pod-privileged-policy)
```

Comment:
This policy has been rewritten with rust in a recent past. As far as I can remember, we do not have assembly script policies anymore.

Reply:
Thanks @jvanz . Ok, so to be clear. KW supports AssembyScript as it produces Wasm binaries. But, we don't have any AS language policies? Are you aware of any at all out there? 

Moving forward, just reduce to talking about two, one in Rust and one in Go?

Reply:
Removed reference to AS for now.